docs/user-guide/global-configurations/okta.md

```diff
@@ -0,0 +1,63 @@
+# Example - Okta SSO
+
+## Prerequisites
+
+A verified account on [Okta](https://www.okta.com/). Okta activates your account only if email verification is successful.
+
+Here's a reference guide to set up your Okta org and application: [Link](https://developer.okta.com/docs/guides/oie-embedded-common-org-setup/go/main/)
+
+## Tutorial
+
+{% embed url="https://www.youtube.com/watch?v=i-7IWkg6Ipk" caption="Okta App Setup" %}
+
+## Steps on Okta Admin Console
+
+Once your Okta org is set up, create an app integration on Okta to get a Client ID and Client Secret.
+
+1. In the Admin Console, go to **Applications** → **Applications**.
+
+2. Click **Create App Integration**.
+
+3. Select **OIDC - OpenID Connect** as the **Sign-in method**.
+
+{% hint style="info" %}
+OIDC stands for OpenID Connect. [Click here](https://www.okta.com/openid-connect/) to read more.
+{% endhint %}
+
+4. Select an application type (Web, Single Page, or Native), and click **Next**.
```

Comment:
Also better to guide user to select an option maybe single page in this case than asking to select any from available 3